version.cmake

```diff
@@ -102,31 +102,45 @@ MACRO(VERSION_COMPUTE)
     # Check if git describe worked and store the returned version number.
     IF(GIT_DESCRIBE_RESULT)
       MESSAGE(
-	"Warning: failed to compute the version number,"
-	" 'git describe' failed:\n"
-	"\t" ${GIT_DESCRIBE_ERROR})
+        "Warning: failed to compute the version number,"
+        " 'git describe' failed:\n"
+        "\t" ${GIT_DESCRIBE_ERROR})
```

Comment:
Since you are at it, would you mind changing this to:
```cmake
      MESSAGE(AUTHOR_WARNING
        "failed to compute the version number,"
        " 'git describe' failed:\n"
        "\t" ${GIT_DESCRIBE_ERROR})
```